diff --git a/Common/cpp/worklets/WorkletRuntime/WorkletRuntime.h b/Common/cpp/worklets/WorkletRuntime/WorkletRuntime.h
index 17cf8f97dac2b3cd720452d5df1ad4523872af3b..770c88993cdd905f0f892e5394cf305d7271bdce 100644
--- a/Common/cpp/worklets/WorkletRuntime/WorkletRuntime.h
+++ b/Common/cpp/worklets/WorkletRuntime/WorkletRuntime.h
@@ -172,21 +172,21 @@ class WorkletRuntime : public jsi::HostObject, public std::enable_shared_from_th
 
   /* #region deprecated */
 
-  /** @deprecated Use `runSync` instead. */
+  /** Deprecated: Use `runSync` instead. */
   template <typename... Args>
   jsi::Value runGuarded(const std::shared_ptr<SerializableWorklet> &worklet, Args &&...args) const {
     return runSync(worklet, std::forward<Args>(args)...);
   }
 
-  /** @deprecated Use `schedule` instead. */
+  /** Deprecated: Use `schedule` instead. */
   void runAsyncGuarded(const std::shared_ptr<SerializableWorklet> &worklet);
 
-  /** @deprecated Use `runSyncSerialized` and extract to `jsi::Value` with
+  /** Deprecated: Use `runSyncSerialized` and extract to `jsi::Value` with
    * `extractSerializableOrThrow` instead. */
   jsi::Value executeSync(jsi::Runtime &rt, const jsi::Value &worklet) const;
-  /** @deprecated Use `runSync` instead. */
+  /** Deprecated: Use `runSync` instead. */
   jsi::Value executeSync(std::function<jsi::Value(jsi::Runtime &)> &&job) const;
-  /** @deprecated Use `runSync` instead. */
+  /** Deprecated: Use `runSync` instead. */
   jsi::Value executeSync(const std::function<jsi::Value(jsi::Runtime &)> &job) const;
 
   /* #endregion */
diff --git a/apple/worklets/apple/AssertTurboModuleManagerQueue.h b/apple/worklets/apple/AssertTurboModuleManagerQueue.h
index b5c99c35e9d55838ff4a80e914dce9f858f49922..3d4af46e3c39aa3c406309636f66a3a5a8d09afd 100644
--- a/apple/worklets/apple/AssertTurboModuleManagerQueue.h
+++ b/apple/worklets/apple/AssertTurboModuleManagerQueue.h
@@ -2,7 +2,7 @@
 
 constexpr auto turboModuleManagerQueueLabel = "com.meta.react.turbomodulemanager.queue";
 
-static bool IsTurboModuleManagerQueue() {
+static inline bool IsTurboModuleManagerQueue() {
   const auto currentQueueLabel = dispatch_queue_get_label(DISPATCH_CURRENT_QUEUE_LABEL);
   return strcmp(currentQueueLabel, turboModuleManagerQueueLabel) == 0;
 }
diff --git a/apple/worklets/apple/AssertJavaScriptQueue.h b/apple/worklets/apple/AssertJavaScriptQueue.h
--- a/apple/worklets/apple/AssertJavaScriptQueue.h
+++ b/apple/worklets/apple/AssertJavaScriptQueue.h
@@ -3,12 +3,12 @@
 // Copied from RCTJSThreadManager.mm
 static NSString *const RCTJSThreadName = @"com.facebook.react.runtime.JavaScript";
 
-static BOOL IsJavaScriptQueue()
+static inline BOOL IsJavaScriptQueue(void)
 {
   return [NSThread.currentThread.name isEqualToString:RCTJSThreadName];
 }
 
-static void AssertJavaScriptQueue()
+static inline void AssertJavaScriptQueue(void)
 {
   react_native_assert(IsJavaScriptQueue() && "This function must be called on the JavaScript queue");
 }
