diff --git a/ReactCommon/react/renderer/mounting/ShadowViewMutation.h b/ReactCommon/react/renderer/mounting/ShadowViewMutation.h
index a34f9654b7eba3a2b37e5bb7ddf5153b1d94ab56..8439dbff412ff67f9044613019c67a16e570560a 100644
--- a/ReactCommon/react/renderer/mounting/ShadowViewMutation.h
+++ b/ReactCommon/react/renderer/mounting/ShadowViewMutation.h
@@ -43,7 +43,7 @@ struct ShadowViewMutation final {
 
   /**
    * Creates and returns an `Insert` mutation.
-   * @deprecated Pass parentTag instead of parentShadowView.
+   * Deprecated: Pass parentTag instead of parentShadowView.
    */
   static ShadowViewMutation InsertMutation(const ShadowView &parentShadowView, ShadowView childShadowView, int index)
   {
@@ -57,7 +57,7 @@ struct ShadowViewMutation final {
 
   /**
    * Creates and returns a `Remove` mutation.
-   * @deprecated Pass parentTag instead of parentShadowView.
+   * Deprecated: Pass parentTag instead of parentShadowView.
    */
   static ShadowViewMutation RemoveMutation(const ShadowView &parentShadowView, ShadowView childShadowView, int index)
   {
@@ -71,7 +71,7 @@ struct ShadowViewMutation final {
 
   /**
    * Creates and returns an `Update` mutation.
-   * @deprecated Pass parentTag instead of parentShadowView.
+   * Deprecated: Pass parentTag instead of parentShadowView.
    */
   static ShadowViewMutation
   UpdateMutation(ShadowView oldChildShadowView, ShadowView newChildShadowView, const ShadowView &parentShadowView)
diff --git a/scripts/react-native-xcode.sh b/scripts/react-native-xcode.sh
index 1dcdf55e190221c9b121df90452dd17ad47ff67b..2e071b7539e556256e1eefb7759e65f146262ab4 100755
--- a/scripts/react-native-xcode.sh
+++ b/scripts/react-native-xcode.sh
@@ -176,7 +176,11 @@ else
   if [[ $EMIT_SOURCEMAP == true ]]; then
     EXTRA_COMPILER_ARGS="$EXTRA_COMPILER_ARGS -output-source-map"
   fi
-  "$HERMES_CLI_PATH" -emit-binary -max-diagnostic-width=80 $EXTRA_COMPILER_ARGS -out "$DEST/$BUNDLE_NAME.jsbundle" "$BUNDLE_FILE"
+  EXTRA_COMPILER_GLOBALS=()
+  if [[ -n "${HERMES_GLOBALS_FILE:-}" ]]; then
+    EXTRA_COMPILER_GLOBALS+=("-include-globals=$HERMES_GLOBALS_FILE")
+  fi
+  "$HERMES_CLI_PATH" -emit-binary -max-diagnostic-width=80 $EXTRA_COMPILER_ARGS "${EXTRA_COMPILER_GLOBALS[@]}" -out "$DEST/$BUNDLE_NAME.jsbundle" "$BUNDLE_FILE"
   if [[ $EMIT_SOURCEMAP == true ]]; then
     HBC_SOURCEMAP_FILE="$DEST/$BUNDLE_NAME.jsbundle.map"
     "$NODE_BINARY" "$COMPOSE_SOURCEMAP_PATH" "$PACKAGER_SOURCEMAP_FILE" "$HBC_SOURCEMAP_FILE" -o "$SOURCEMAP_FILE"
