diff --git a/apple/reanimated/apple/REAAssertTurboModuleManagerQueue.h b/apple/reanimated/apple/REAAssertTurboModuleManagerQueue.h
index 46e41f6ce333710d9b240aa5bc00a530a13ca216..0f786b71dc529ed1981e82e4c73b89a8695842ad 100644
--- a/apple/reanimated/apple/REAAssertTurboModuleManagerQueue.h
+++ b/apple/reanimated/apple/REAAssertTurboModuleManagerQueue.h
@@ -2,7 +2,7 @@
 
 constexpr auto turboModuleManagerQueueLabel = "com.meta.react.turbomodulemanager.queue";
 
-static bool REAIsTurboModuleManagerQueue() {
+static inline bool REAIsTurboModuleManagerQueue() {
   const auto currentQueueLabel = dispatch_queue_get_label(DISPATCH_CURRENT_QUEUE_LABEL);
   return strcmp(currentQueueLabel, turboModuleManagerQueueLabel) == 0;
 }
diff --git a/apple/reanimated/apple/REANodesManager.h b/apple/reanimated/apple/REANodesManager.h
index 4646e125ab0a6ce1e3cb4701130d34c016e93cde..22065c6a48d72c7e441040930ea4112902e5adc9 100644
--- a/apple/reanimated/apple/REANodesManager.h
+++ b/apple/reanimated/apple/REANodesManager.h
@@ -3,6 +3,8 @@
 
 #import <reanimated/apple/READisplayLink.h>
 
+NS_ASSUME_NONNULL_BEGIN
+
 typedef void (^REAOnAnimationCallback)(READisplayLink *displayLink);
 typedef void (^REAEventHandler)(id<RCTEvent> event);
 typedef void (^CADisplayLinkOperation)(READisplayLink *displayLink);
@@ -10,7 +12,7 @@ typedef void (^REAPerformOperations)();
 
 @interface REANodesManager : NSObject
 
-@property (weak) RCTSurfacePresenter *surfacePresenter;
+@property (weak, nullable) RCTSurfacePresenter *surfacePresenter;
 
 - (nonnull instancetype)init;
 - (void)invalidate;
@@ -23,3 +25,5 @@ typedef void (^REAPerformOperations)();
 - (void)maybeFlushUIUpdatesQueue;
 
 @end
+
+NS_ASSUME_NONNULL_END
